torchvision/ops/_custom_ops.py

```diff
@@ -0,0 +1,46 @@
+import os
+import sys
+import torch
+
+
+# load the custom_op_library and register the custom ops
+lib_dir = os.path.join(os.path.dirname(__file__), '..')
+extension = os.path.basename(torch._C.__file__).rsplit('.', 1)[1]
+custom_op_lib = os.path.join(lib_dir, 'custom_ops.' + extension)
+torch.ops.load_library(custom_op_lib)
```

Comment:
Instead of renaming the lib to `custom_ops` in the build, what about using something similar to https://github.com/pytorch/pytorch/blob/d95763b4dcf9d610232c7d02a7d6bdacc423562c/torch/utils/cpp_extension.py#L1046-L1054 from PyTorch to simplify this process?

Something like

```python
import imp
file, path, description = imp.find_module("custom_ops", [lib_dir])
torch.ops.load_library(path)
```

Also, I think we should consider renaming `custom_ops` to start with an underscore?